shardingsphere-kernel/shardingsphere-transaction/shardingsphere-transaction-core/src/main/java/org/apache/shardingsphere/transaction/core/TransactionType.java

```diff
@@ -22,7 +22,7 @@
  */
 public enum TransactionType {
     
-    LOCAL, XA, BASE;
+    LOCAL, XA, BASE, MANUALXA;
```

Comment:
I'd like to suggest to change ```XA``` to ```JTA``` but it should be fixed in another PR in the future.

Reply:
yes, how about JTAXA?

Reply:
```JTAXA``` is weird but I think we can discuss it later.